app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copied_file, new_file = command.split()[1:]
```

Comment:
if u don't need some value while unpacking it is more popular way to use `_`
f.e. `_, var1, var2 = command.split()`